src/frontend/komponenter/Fagsak/Vedtak/AvsnittSkjema.tsx

```diff
@@ -73,17 +58,16 @@ const AvsnittSkjema: React.FC<IProps> = ({
     }, [avsnitt]);
 
     return (
-        <>
-            <StyledEkspanderbartpanel
-                tittel={avsnitt.overskrift ? avsnitt.overskrift : ''}
-                apen={!erLesevisning && åpen}
-                className={
-                    !erLesevisning && harPåkrevetFritekstMenIkkeUtfylt
-                        ? 'panelMedGulmarkering'
-                        : 'panel'
```

Comment:
TODO: fikse styling av border left hvis fritekst er påkrevd men ikke utfylt

Reply:
Nå
![image](https://github.com/navikt/familie-tilbake-frontend/assets/25459913/ec3f3d0d-e8ad-4d68-ac93-12a6de183707)

Reply:
Har fikset mellomrommet mellom periodene etter at dette bildet ble tatt (se de andre eksemplene fra denne komponenten, uten kantlinjen, for riktig mellomrom)